UDV-Core/examples/Vilo3D/index.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+    <title>UDV - Vilo3D Test</title>
+    <link rel="stylesheet" href="/src/css/styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <div id="viewerDiv"></div>
+    <div id="info">UDV v0.3</div>
+    <div id="cartoDataSource">données urbaines : https://data.grandlyon.com/</div>
+    <img id="logoIMU" src="/img/logo-imu.png"></img>
+    <img id="logoLIRIS" src="/img/logo-liris.png"></img>
+    <script src="../../dist/udvcore.js"></script>
+    <script src="../../dist/udvcore.js"></script>
```

Comment:
The line `<script src="../../dist/udvcore.js"></script> ` is duplicated.

Reply:
Yep. Fixed.

Reply:
Yep. Fixed.